### PR TITLE
Webex Sink Integration

### PR DESCRIPTION
**Pull request by alikhanxgrid**
### Issue

[New Sink: Webex](https://github.com/robusta-dev/robusta/issues/343)

### Description

This PR adds a integrates Webex as a sink to Robusta. Main logic of sending Findings to Robusta is in sender.py file. The docs/ folder contain the instructions for configuring Webex with your Robusta deployment. 

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes (if applicable)
- [x] Every function, interface, class has a comment describing what it does and input/output parameters

### CODE REVIEW

File: src/robusta/integrations/webex/sender.py [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+import os
+
+
+INVESTIGATE_ICON = u"\U0001F50E"
+SILENCE_ICON = u"\U0001F515"
+
+MAX_BLOCK_CHARS = 7439
+
+
+class WebexSender:
```
Comment (abdullah-shah1) [inline src/robusta/integrations/webex/sender.py:34]: pls add docs.

File: src/robusta/integrations/webex/sender.py [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        message, table_blocks, file_blocks, description = self._seperate_blocks(
+            finding, platform_enabled)
+
+        pdf = None
+        if file_blocks:
+            pdf = self._create_pdf(file_blocks)
+
+        if table_blocks:
+            if pdf:
+                pdf.output("file.pdf", "F")
```
Comment (abdullah-shah1) [inline src/robusta/integrations/webex/sender.py]: better name ?

File: src/robusta/integrations/webex/sender.py [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+            pdf.add_page()
+            file_type = "Photo" if is_image(blocks.filename) else "Document"
+            if file_type is "Document":
+                # insert the texts in pdf
+                contents = blocks.contents.decode("utf-8")
+                # create a cell
+                pdf.cell(200, 10, txt=f"{blocks.filename}\n",
+                         ln=1, align='C')
+                for text in contents:
+                    pdf.write(5, text)
```
Comment (abdullah-shah1) [inline src/robusta/integrations/webex/sender.py]: why 5 ? lets put some comments around magic numbers/strings.

File: src/robusta/integrations/webex/sender.py [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+            return message_content, table_blocks, file_blocks, description
+
+    def _create_pdf(self, file_blocks: List[FileBlock]):
+        pdf = FPDF()
+        pdf.set_font("Arial", size=15)
+        for blocks in file_blocks:
+            # Add a page
+            pdf.add_page()
+            file_type = "Photo" if is_image(blocks.filename) else "Document"
+            if file_type is "Document":
```
Comment (abdullah-shah1) [inline src/robusta/integrations/webex/sender.py]: magic

**Review by abidiqbalxgrid — APPROVED** [review]

**Review by yousaf-aftab — APPROVED** [review]

File: src/robusta/integrations/webex/sender.py (hunk 1 above)
Comment (alikhanxgrid) [reply src/robusta/integrations/webex/sender.py:34]: Added comments and docs.

File: src/robusta/integrations/webex/sender.py (hunk 3 above)
Comment (alikhanxgrid) [reply src/robusta/integrations/webex/sender.py]: Added

File: src/robusta/integrations/webex/sender.py (hunk 4 above)
Comment (alikhanxgrid) [reply src/robusta/integrations/webex/sender.py]: Resolved. Assigned strings to variables

**Pull request merged**